Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Compact/BasicOperators/CompactDivideNodeable.ScriptCanvasNodeable.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<ScriptCanvas Include="Include/ScriptCanvas/Libraries/Compact/BasicOperators/CompactDivideNodeable.h" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+			  xsi:noNamespaceSchemaLocation="../../../AutoGen/ScriptCanvasNodeable.xsd">
+    <Class Name="CompactDivideNodeable"
+        QualifiedName="Nodeables::CompactDivideNodeable"
+        PreferredClassName="/="
+        Category="Compact/Basic Operators"
+        Namespace="ScriptCanvas"
+        Description="Divides first input number by the second input number"
```

Comment:
Should we instead of using `first` and `second`, use `parameter 'a'` and `parameter 'b'`?
```suggestion
        Description="Divides parameter 'a' by parameter 'b'"
```

Reply:
This comment applies to all of the related node descriptions

Reply:
I am worried that using parameter `a` and `b` instead of `first` and `second` might be a little confusing since the labels for the inputs are hidden by default.

Reply:
Well, if the labels are hidden, then 'a' and 'b' won't make sense then.